libkpod/remove.go

```diff
@@ -17,6 +17,9 @@ func (c *ContainerServer) Remove(container string, force bool) (string, error) {
 	ctrID := ctr.ID()
 
 	cState := c.runtime.ContainerStatus(ctr)
+	if cState.Status == oci.ContainerStatePaused && !force {
+		return "", errors.Errorf("cannot remove paused container %s", ctrID)
+	}
 	if cState.Status == oci.ContainerStateCreated || cState.Status == oci.ContainerStateRunning {
```

Comment:
Nit: could this be an else-if? It looks like it and the checks that come after don't overlap.

Reply:
Perhaps a switch statement.

Reply:
Switched to a Switch.